src/components/Form.js

```diff
@@ -29,54 +29,77 @@ const species = ['Black Bear', 'Bobcat', 'Cougar/Mountain Lion', 'Coyote', 'Opos
 class Form extends Component {
   state = {
     species: '',
-    timestamp: '',
+    timestamp: new Date(),
+    mapLat: 47.608013,
+    mapLng: -122.335167,
   };
 
   handleChange = event => {
     this.setState({ [event.target.name]: event.target.value });
   };
 
+  getMapCoordinates = dataFromMap => {
+    this.setState({ mapLat: dataFromMap.lat, mapLng: dataFromMap.lng });
+  };
+
+  handleTimestampChange = timestamp => {
+    this.setState({
+      timestamp: new Date(timestamp)
+    });
+  };
+
   render() {
     const { classes } = this.props;
+    const { mapLat, mapLng, timestamp } = this.state;
+
     return (
       <form className="formWizardBody" autoComplete="off">
-        <div>
+        <div className="formItem">
           <h4>Which animal did you see?</h4>
           <FormControl className={classes.formControl}>
-            <InputLabel htmlFor="species">Species</InputLabel>
-            <Select
+            <TextField
               value={this.state.species}
+              select
+              variant="outlined"
+              label="Species"
               onChange={this.handleChange}
               inputProps={{
                 name: 'species',
                 id: 'species',
               }}
             >
               {species.map((type, idx) => <MenuItem key={idx} value={type}>{type}</MenuItem>)}
-            </Select>
+            </TextField>
           </FormControl>
         </div>
-        <div>
 
+        <div className="formItem">
```

Comment:
This is more of a me-learning question: when do you use a built-in tag like `<div>` vs a material-ui component like `<Grid>`?

Reply:
So yeah, it really depends on what you're trying to build. Ultimately, Material UI doesn't have a nice form component like Antd (i.e. no real Form Items) so I just used a `<div>` to wrap each field in the form. I try to use the pre-defined UI kit components where ever possible because they make life super simple.

Reply:
Cool, that makes a lot of sense. Thank you!